src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Don't run .NET Framework only tests on non Windows. -->
+    <SkipTests Condition="'$(OS)' != 'Windows_NT'">true</SkipTests>
+    <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPartialFacadeAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\System\MutableDecimal.cs" Link="Common\System\MutableDecimal.cs" />
+    <Compile Include="$(CommonPath)tests\System\Buffers\NativeMemoryManager.cs" Link="Common\System\Buffers\NativeMemoryManager.cs" />
+    <Compile Include="$(CommonPath)tests\System\Diagnostics\Tracing\TestEventListener.cs" Link="Common\System\Diagnostics\Tracing\TestEventListener.cs" />
+    <Compile Include="$(CommonPath)tests\TestUtilities\System\AssertExtensions.cs" Link="Common\System\AssertExtensions.cs" />
+    <Compile Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.cs" Link="Common\System\PlatformDetection.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
+    <Compile Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Unix.cs" Link="Common\System\PlatformDetection.Unix.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <Compile Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Windows.cs" Link="Common\System\PlatformDetection.Windows.cs" />
+  </ItemGroup>
```

Comment:
Same comment as somewhere above, this should be moved into the root Directory.Build.targets file and only apply for test projects.